### PR TITLE
[MRG] Feature posts by author and other fixes

### LINKED ISSUE #125: Unify all date formats
It's wrong on tag pages

### PR DESCRIPTION
**Pull request by aapeliv**
Closes #125 and #119

### CODE REVIEW

File: src/helpers/AuthorHelpers.js [hunk 1]
```diff
@@ -21,18 +21,17 @@ Given a set of authors, generates a line like this with links:
 */
 export function authorAndDateLine(authors, date) {
```
Comment (shuttle1987) [inline src/helpers/AuthorHelpers.js]: Functions like this I'd love to have some tests for

File: src/templates/tag.js [hunk 2]
```diff
@@ -8,18 +8,12 @@ import PostListing from "../components/PostListing/PostListing";
 export default class TagTemplate extends React.Component {
   render(){
     const tag = this.props.pathContext.tag;
-    const postEdges = this.props.data.allMarkdownRemark.edges;
-    const numberOfPosts = this.props.data.allMarkdownRemark.totalCount;
-    const allAuthors = this.props.data.authors.edges;
-    const filteredPosts = postEdges.filter(postEdges => postEdges.node.frontmatter.draft !== true);
```
Comment (shuttle1987) [inline src/templates/tag.js]: Given that this has been removed is this still going to hide draft posts?

Comment (shuttle1987) [reply src/templates/tag.js]: I feel like this should likely be in the graphql query, and that we should use query fragments so we don't do the same thing everywhere

Comment (aapeliv) [reply src/templates/tag.js]: I think you're right. This might load drafts into the client, which we obviously don't want...

**Comment by shuttle1987** [comment]
Looks good now!

**Pull request merged**